lib/src/source/hosted.dart

```diff
@@ -1100,3 +1128,82 @@ class _RefAndCache {
   @override
   bool operator ==(Object other) => other is _RefAndCache && other.ref == ref;
 }
+
+const checksumHeaderName = 'x-goog-hash';
```

Comment:
Does this need to be exported?
```suggestion
const _checksumHeaderName = 'x-goog-hash';
```

Reply:
So, that export is used in "test/package_server.dart":

https://github.com/dart-lang/pub/blob/ce3a0ec5fd8598d609b6c4498b20ceb37c6798ae/test/package_server.dart#L122-L128

https://github.com/dart-lang/pub/blob/ce3a0ec5fd8598d609b6c4498b20ceb37c6798ae/test/package_server.dart#L282-L283

Reply:
If it needs to be exported only for testing you can mark it `@visibleForTesting`

Reply:
Nice! Added that.